roles/ceph-grafana/files/dashboards/ceph-cluster.json

```diff
@@ -51,7 +51,7 @@
         "rgba(237, 129, 40, 0.9)",
         "rgb(255, 0, 0)"
       ],
-      "datasource": "$datasource",
```

Comment:
FYI: This was intentional, with `$datasource` here, the datasource was actually selectable in grafana. We should probably drop the dashboards from the PR altogether and use those that are provided in ceph's `ceph-grafana-dashboards` package. Having them as part of the deployment was sort of a hack for luminous where these dashboards did not exist upstream, yet. Alternatively, we can have them as back-up in case the package does not exist (that was my original plan, the code should be ready for this, it just does not install the package, yet).

Reply:
Oh, okay - it's just that none of the dashboards were working by default. I'll update the PR to use the packaged dashboards. It does look like they use a templated datasource name as well, so I'll see about fixing the default behavior.

I'd tentatively suggest that we shouldn't store them in two places, for maintenance reasons. It shouldn't be hard to backport the dashboards to luminous if we need to.

Reply:
It was a downstream hack so we probably don't need to care about it upstream and we can just drop the dashboards and install the package instead. The official dashboard v2 support is dedicated for nautilus+.

It seems weird that they did not work, what error did you get? What grafana version were you using? Did you by any chance hit this bug? https://github.com/grafana/grafana/issues/14239

Reply:
It was/is v5.4.2 - I actually can't remember if it produced an error - the main symptom was blank graphs. Once I get the purge playbook working I'll tear down my current setup and dig a little deeper.

Reply:
Update: I switched to using the packaged dashboards. It looks like the failure in datasource name templating only happens when visiting grafana directly without being signed in; once I logged in as admin, they worked properly. When viewing from the dashboard, they worked as well (even before signing in on the other grafana tab).